Formula/maxwell.rb

```diff
@@ -0,0 +1,39 @@
+class Maxwell < Formula
+  desc "Maxwell's daemon, a mysql-to-json kafka producer"
+  homepage "http://maxwells-daemon.io/"
+  url "https://github.com/zendesk/maxwell/releases/download/v1.13.1/maxwell-1.13.1.tar.gz"
+  sha256 "3ab7489d47f2ec04677122c8786cf573572b9bbb511b184802d2dcfe82e83781"
+
+  bottle do
+    cellar :any
+    sha256 "3ab7489d47f2ec04677122c8786cf573572b9bbb511b184802d2dcfe82e83781" => :high_sierra
+    sha256 "3ab7489d47f2ec04677122c8786cf573572b9bbb511b184802d2dcfe82e83781" => :sierra
+    sha256 "3ab7489d47f2ec04677122c8786cf573572b9bbb511b184802d2dcfe82e83781" => :el_capitan
+  end
+
+  depends_on "maven" => :build
+  depends_on :java => "1.8"
+
+  pour_bottle? do
```

Comment:
Use `bottle :unneeded` above, since `def install` is only moving files around rather than doing any sort of building/compile process. This block can disappear entirely.

Reply:
Removed the whole block.